download/releaseNotes/releaseNotes8.adoc

```diff
@@ -32,6 +32,19 @@ You can now modify your streams using link:https://docs.optaplanner.org/latestFi
 We have made some tweaks under the hood
 so that your experience with the recently released link:https://openjdk.java.net/projects/jdk/16/[OpenJDK 16] continues to be smooth.
 
+=== Inject and Autowire ConstraintVerifier in Quarkus and Spring Boot
+
+You can now link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsTestingQuarkus[inject the Constraint Verifier in Quarkus] and
+link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsTestingSpringBoot[autowire the Constraint Verifier in Spring Boot], allowing
+you to easily test your link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreams[ConstraintProvider constraints].
+
+=== Faster Domain Accessors and Cloning with Gizmo
+
+We have added link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#domainAccess[Gizmo generated domain accessors and solution
+cloners], which offer better performance than the reflection based
+domain accessors and solution cloners but come with some
+restrictions.
```

Comment:
I don't think we should do cliffhangers.
Say something like "when used with Quarkus".

I wouldn't really mention the "everything public" option, as that is an antipattern.

Reply:
I could remove "but come with some restrictions" entirely; I don't like "when used with Quarkus" since it make it seems like only Quarkus get a benefit/it require Quarkus.